src/stylesheets/components/_button-groups.scss

```diff
@@ -0,0 +1,102 @@
+// Default styles
+.usa-button-group {
+  @include unstyled-list;
+  display: flex;
```

Comment:
Using `display: flex` here allows block elements to display in single row without the the whitespace that gets added when using `display: inline-block`